core/src/components/checkbox/checkbox.tsx

```diff
@@ -328,7 +328,7 @@ export class Checkbox implements ComponentInterface {
             {this.renderHintText()}
           </div>
           <div class="native-wrapper">
-            <svg class="checkbox-icon" viewBox="0 0 24 24" part="container">
+            <svg class="checkbox-icon" viewBox="0 0 24 24" part="container" aria-hidden="true" role="presentation">
```

Comment:
```suggestion
            <svg class="checkbox-icon" viewBox="0 0 24 24" part="container" aria-hidden="true">
```

We shouldn't need `role` since it is removed from the accessibility tree by using `aria-hidden`